.github/workflows/ci-plugin-server.yml

```diff
@@ -36,146 +36,6 @@ jobs:
             - name: Lint with ESLint
               run: yarn lint
 
-    tests-postgres-1:
```

Comment:
I don't think this is safe to drop. This will make some legit tests not run I believe

We shouldn't include this in this PR and instead handle in a separate, plugin-server specific PR.